dvc/repo/metrics/show.py

```diff
@@ -171,20 +172,21 @@ def _collect_metrics(repo, path, recursive, typ, xpath, branch):
             - xpath:
     """
     outs = [out for stage in repo.stages() for out in stage.outs]
+    found = []
 
-    if path:
-        try:
-            outs = repo.find_outs_by_path(path, outs=outs, recursive=recursive)
-        except OutputNotFoundError:
-            logger.debug(
```

Comment:
why did you remove debug message though?

Reply:
:thinking: thought you said we don't need this anymore.
https://github.com/iterative/dvc/pull/2319#discussion_r310354892

Reply:
my bad, then, I'll bring it back

Reply:
@mroutis I meant that `return []` doesn't make sense, because we should try to process all targets.